stl/inc/yvals_core.h

```diff
@@ -1144,7 +1144,17 @@
 #define _CXX20_DEPRECATE_IS_ALWAYS_EQUAL
 #endif // ^^^ warning disabled ^^^
 
-// next warning number: STL4034
+#if _HAS_CXX20 && !defined(_SILENCE_CXX20_CISO646_REMOVED_WARNING) && !defined(_SILENCE_ALL_CXX20_DEPRECATION_WARNINGS)
+#define _CXX20_REMOVE_CISO646                                             \
+    [[deprecated("warning STL4036: "                                      \

```

Comment:
https://github.com/microsoft/STL/pull/2583 uses `STL4034` and `STL4035`.

Reply:
We're going to have merge conflicts anyway since the PRs add code in the same place; we may as well keep them independent and avoid gaps in our numbering by using `STL4034` here. We'll have to fix the second PR to be applied in any case.

Reply:
ok, I'll change to `STL4034`